.pre-commit-config.yaml

```diff
@@ -1,13 +1,32 @@
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+        - id: check-added-large-files
+        - id: check-case-conflict
+        - id: check-json
+        - id: check-merge-conflict
+        - id: check-symlinks
+        - id: check-toml
+        - id: check-yaml
+        - id: detect-private-key
+        - id: end-of-file-fixer
+        - id: trailing-whitespace
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        name: isort (python)
```

Comment:
[Documentatiuon] says
```
    hooks:
      - id: isort
        args: ["--profile", "black", "--filter-files"]
```

Reply:
I think I got it from [here](https://pycqa.github.io/isort/docs/configuration/pre-commit.html) where no extra options are set, but [here](https://pycqa.github.io/isort/docs/configuration/black_compatibility.html) it has your syntax for black compatibility specifically, so indeed we should use that.

Reply:
```suggestion
        args: ["--profile", "black", "--filter-files"]
        name: isort (python)
```